gamemodes/terrortown/gamemode/shared/huds/pure_skin/cl_init.lua

```diff
@@ -50,6 +52,28 @@ function HUD:Initialize()
 	BaseClass.Initialize(self)
 end
 
+---
+-- This function will return a table containing all keys that will be stored by
+-- the @{HUD:SaveData} function.
+-- @return table
+-- @realm client
+function HUD:GetSavingKeys()
+	if not savingKeys then
```

Comment:
there is no need to store this / cache this

Reply:
just saw, that we also do this in the scalable hud, dunno why though :D i think this can be removed too :D

Reply:
I think I understand what you mean and will change that.